components/x-teaser/src/Image.jsx

```diff
@@ -17,20 +17,24 @@ const aspectRatio = ({ width, height }) => {
 	return null;
 };
 
-const NormalImage = ({ src }) => (
-	<img className="o-teaser__image" src={src} alt="" />
+const NormalImage = ({ src, srcSet }) => (
+	<img className="o-teaser__image" src={src} srcSet={srcSet} alt="" />
 );
 
-const LazyImage = ({ src, lazyLoad }) => {
+const LazyImage = ({ src, srcSet, lazyLoad }) => {
 	const lazyClassName = typeof lazyLoad === 'string' ? lazyLoad : '';
-	return <img className={`o-teaser__image ${lazyClassName}`} data-src={src} alt="" />;
+	return <img className={`o-teaser__image ${lazyClassName}`} data-src={src} srcSet={srcSet} alt="" />;
```

Comment:
Could we end up loading one of the `srcSet` images here even though they should be lazy loaded? Should this also be a data attribute that gets swapped when it's coming into the viewport?